app/views/users/edit.html.erb

```diff
@@ -1,6 +1,12 @@
+<% content_for :navbar_links do %>
+  <%= render 'layouts/basic_navbar/user_links' %>
+<% end %>
+
+<%= render "layouts/page_heading", locals = {heading: "Edit Profile: #{current_user.nickname"}%>
```

Comment:
This is backwards the link above (new business path)

above its => "mhecker: New Business"
Here its => "Edit Profile: mhecker"

Let's make those match.